File: cpp/bmi.cpp


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	float wzrost;
    int waga;
    float bmi;
    
    cout<<"Podaj swoją masę: ";
    cin>>waga;
    cout<<"Podaj swój wzrost(w metrach): ";
    cin>>wzrost;
    
    bmi = waga/(wzrost*wzrost);
    
   if (bmi<18.5)
    {
        cout<<"masz mały problem";
    }
    else if (bmi<25)
    {
        cout<<"spoko";
    }
    else if (bmi<30)
    {
        cout<<"masz większy problem";
    }
    else if (bmi>=30)
    {
        cout<<"jesteś grubasem";
    }


	return 0;
}

File: python/bmi.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def main(args):



    wzrost = float(input("Podaj twój wzrost (w metrach): "))
    masa = int(input("Podaj swóją masę (w kg): "))
    bmi = masa / (wzrost * wzrost)
    if bmi < 18.5:
        print('Masz mały problem')
    elif bmi < 25:
        print('Spoko')
    elif bmi < 30:
        print('Masz większy problem')
    elif bmi > 30:
        print('Jesteś za gruby')

#        wzrost = float(input("Podaj twój wzrost (w metrach): "))
#        masa = int(input("Podaj swóją masę (w kg): "))
#        bmi = masa / (wzrost * wzrost)
    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
